lib/expectedConditions.ts

```diff
@@ -404,8 +403,8 @@ export class ProtractorExpectedConditions {
   }
 
   /**
-   * An expectation for checking that an element is either invisible or not
-   * present on the DOM. This is the opposite of 'visibilityOf'.
+   * An expectation for checking that an element is invisible on the DOM.
```

Comment:
This should still have "either invisible or not present on the DOM". If `invisibilityOf` returns true, that means `visibility` returned false. So either `presenceOf` is false or `isDisplayed` is false.